package.json

```diff
@@ -24,6 +24,7 @@
     "axios": "^0.21.1",
     "bignumber.js": "^9.0.1",
     "bootstrap": "^4.5.3",
+    "chart.js": "^2.9.4",
```

Comment:
Is this used anywhere?

Reply:
vue-chart.js requires it (apparently not added to internal deps)
https://vue-chartjs.org/guide/#installation